packages/compile-solidity/profiler/shouldIncludePath.js

```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+function shouldIncludePath(filePath) {
+  return path.extname(filePath) !== ".vy";
+}
```

Comment:
Just to make this more extensible.  Idk, feel free to disregard.
```javascript
function createExtensionCheck(extension = ".vy") {
  return function shouldIncludePath(filePath) {
    return path.extname(filePath) !== extension;
  }
}
```

Reply:
or 
```javascript
function shouldIncludePath(filePath, extension = ".vy") {
  return path.extname(filePath) !== extension;
}
```

Reply:
Hm, I'm not sure we want to codify that negation like that. That check is really just a code smell that I left in, in case there are unknown warts.